common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -471,7 +471,7 @@ trait FeatureSwitches {
     "When ON show YouTube related video suggestions in YouTube media atoms",
     owners = Seq(Owner.withGithub("siadcock")),
     safeState = Off,
-    sellByDate = new LocalDate(2019, 7, 17),
+    sellByDate = new LocalDate(2019, 7, 25),
```

Comment:
```suggestion
    sellByDate = new LocalDate(2019, 8, 1),
```

This should drop it into the next fun week